table/builder.go

```diff
@@ -281,3 +277,37 @@ func (b *Builder) writeChecksum(data []byte) {
 	_, err = b.buf.Write(buf[:])
 	y.Check(err)
 }
+
+func u32ToBytes(v uint32) []byte {
```

Comment:
We can move these functions to y package so that these can be reused.

Reply:
Moved to y.